### DISCUSSION TITLE
Live Polling: have it enabled by default on pageload

### DISCUSSION THREAD

**Discussion by Benratelade** [sidekiq/sidekiq#5197] [Q&A]
Hi there, 

I was recently updating a bunch of gems in a project and this included a bump for Sidekiq from 6.2.2 to 6.4.1. 

We noticed in our tests that taking users to URLs that include the param `poll=true` doesn't seem to work anymore. Sure enough, the "Live Poll" button no longer points to a URL and (I assume) enables some javascript that handles polling the state of queues. 

My question is this: is there a new and improved way of taking users to a page where "Live Poll" is enabled by default in the newer versions? 

As a follow up question, if there isn't a new way of achieving this, was this an intended change? Or maybe the fact that users could visit a page with poll enabled by default was never a supported feature and only a happy side-effect of how Sidekiq's web interface was built? 

Thanks in advance for anyone who can help shed some light on this.

**Comment by mperham** [comment]
The Web UI's [JS was overhauled in 6.3.0](https://github.com/mperham/sidekiq/blob/main/Changes.md#630). Any loss of functionality should be considered a bug. PRs always welcome. The code is here: https://github.com/mperham/sidekiq/blob/main/web/assets/javascripts/application.js

**Reply by Benratelade** [reply]
Great thank you for taking the time to respond. I'm not sure when but I'll try to have a look at what can be done to restore functionality. 

Thanks again.

**Comment by Benratelade** [comment]
Hi @mperham, 

I put together a very quick solution that handles when ?poll=true, and does nothing with any other params / values. 

See it here: 
https://github.com/Benratelade/sidekiq/tree/enable_live_poll_from_url_params

Let me know if that's not the right way of contributing.

**Reply by mperham** [reply]
The best way is to send a PR with your changes.

**Comment by mperham** [comment]
Please make sure you are testing against main as there have been some JS changes since 6.4.1.
